dynamic-layers/raspberrypi/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -1,9 +1,8 @@
-do_install_append_rpi () {
+do_install_append() {
```

Comment:
please dont change the overrides. This will cause multi BSP build failures.

Reply:
thanks for the feedback. How does it work for multi BSP builds?
you include the layers for each BSP at the same time (activating both dynamic layers in turn), and then when you bitbake a particular image for one machine (e.g. RPI) it will pick the "_rpi" conditional and for the second machine in another bitbake it won't use the "_rpi" conditional?
if that is the case, then probably the recipes in dynamic layers should all account for this conditions.
I wasn't accounting for multi BSP layering :)